.github/workflows/mega-linter.yml

```diff
@@ -0,0 +1,84 @@
+---
+# Mega-Linter GitHub Action configuration file
+# More info at https://nvuillam.github.io/mega-linter
+name: Mega-Linter
+
+on:
+  # Trigger mega-linter at every push. Action will also be visible from Pull Requests to master
+  push: # Comment this line to trigger action only on pull-requests (not recommended if you don't pay for GH Actions)
+  pull_request:
+    branches: [master]
+
+env:
+  # Apply linter fixes configuration
+  APPLY_FIXES: all # Uncomment to apply fixes provided by linters. You can also specify the list of fixing linters
+  APPLY_FIXES_EVENT: pull_request # Decide which event triggers application of fixes in a commit or a PR (pull_request (default), push, all)
+  APPLY_FIXES_MODE: commit # If APPLY_FIXES is used, defines if the fixes are directly committed (commit) or posted in a PR (pull_request)
+
+jobs:
+  build:
+    name: Mega-Linter
+    runs-on: ubuntu-latest
+    steps:
+      # Git Checkout
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      # Mega-Linter
+      - name: Mega-Linter
+        uses: nvuillam/mega-linter@v4
+        env:
+          # All available variables are described in documentation
+          # https://nvuillam.github.io/mega-linter/#configuration
+          VALIDATE_ALL_CODEBASE: true # ${{ github.event_name == 'push'  && github.ref == 'refs/heads/master' }} # Validates all source when push on master, else just the git diff with master. Override with true if you always want to lint all sources
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # DISABLE: COPYPASTE,SPELL # Uncomment to disable copy-paste and spell checks
+          # ADD YOUR CUSTOM ENV VARIABLES HERE
+          FILTER_REGEX_EXCLUDE: "(/test/)"
+          DISABLE_LINTERS: JAVASCRIPT_STANDARD
+
+      # Upload Mega-Linter artifacts
+      - name: Archive production artifacts
```

Comment:
What's the benefit of uploading logs?

Reply:
I created a demo PR: https://github.com/nvuillam/markdown-link-check/pull/2

This was the results I had after the first run in your repo
As you can see, console logs contains a not very user-friendly mix of all linter logs

Uploading artifacts allows to use [TextReporter](https://nvuillam.github.io/mega-linter/reporters/TextReporter/) to have separated log files by linter, much more easy to browse and read :) 

Without artifacts :

![image](https://user-images.githubusercontent.com/17500430/99648691-9e372b80-2a53-11eb-80c6-1d7b953f53b7.png)

With artifacts:

![image](https://user-images.githubusercontent.com/17500430/99648859-d0488d80-2a53-11eb-8268-cfa7be93174a.png)